### PR TITLE
Pass bearer token as a prop

### PR DESCRIPTION
**Pull request by LukasHirt**
If the token is passed as a prop, do not trigger auth flow from oidc js client. Bonus of this PR - add some basic tests and run them in CI.

### CODE REVIEW

**Comment by LukasHirt** [comment]
Agh forgot to add changelog. Will do that after the review in case there'd be some changes requested so I do not have to push multiple times. Will also take a look at the tests then.

**Review by kulmann — APPROVED** [review]
Code looks good to me. Unfortunately I'm not able yet to try it on my machine.

File: src/components/ListResources.vue [hunk 1]
```diff
@@ -21,15 +21,19 @@
       >
         <div v-if="!isLocationPicker">
           <oc-checkbox
-            class="uk-margin-small-left"
+            class="file-picker-resource-checkbox uk-margin-small-left"
```
[the remark this answers is not in the document]
Comment (LukasHirt) [reply src/components/ListResources.vue:24]: The current version of ODS here doesn't have yet those classes. Will update it in separate PR and then change in there :wink:

**Comment by LukasHirt** [comment]
Had to undo the rebase because it deleted all the tests that I've written :shrug: Did a merge commit instead

**Comment by LukasHirt** [comment]
Hmm, need to find out a way how to test the snapshot here. There is a time which is changing. I thought that generating always the current time might help but that doesn't seem to help as well.

**Comment by LukasHirt** [comment]
I removed from the `getCurrentDate` function minutes and seconds. This should hopefully result in always the same result in snpashot.

**Pull request merged**